toolbox/certbot/restart-airflow.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o errexit
```

Comment:
I really like this ability here to have a shell (read: script) bail if any of the commands that are executed fails. 

But in this case, only the docker commands have any meaningful risk of failing. If the `cd` or the `echo` commands don't succeed, we have much bigger problems than our airflow install. 

The docker commands - if the `down` fails, then I feel like it's still prudent to continue on with the `up` operation. 

This is super edge, niche case stuff though. I wouldn't give this more than a seconds thought and hit the resolve button with no regrets for this comment if you want :)